src/BenchmarksApps/Kestrel/PlatformBenchmarks/BenchmarkConfigurationHelpers.cs

```diff
@@ -19,26 +25,42 @@ public static IWebHostBuilder UseBenchmarksConfiguration(this IWebHostBuilder bu
             // Handle the transport type
             var webHost = builder.GetSetting("KestrelTransport");
 
-            if (string.Equals(webHost, "Sockets", StringComparison.OrdinalIgnoreCase))
+            Console.WriteLine($"KestrelTransport={webHost}");
+
+            if (string.Equals(webHost, "SocketPipe", StringComparison.OrdinalIgnoreCase))
             {
-                builder.UseSockets(options =>
+                builder.ConfigureServices(services =>
                 {
-                    if (int.TryParse(builder.GetSetting("threadCount"), out int threadCount))
-                    {
-                       options.IOQueueCount = threadCount;
-                    }
+                    services.AddSingleton<IConnectionListenerFactory, SocketPipeTransportFactory>();
 
-#if NETCOREAPP5_0
-                    options.WaitForDataBeforeAllocatingBuffer = false;
-#endif
+                    services.AddTransient<IConfigureOptions<KestrelServerOptions>, KestrelServerOptionsSetup>();
+                    services.AddSingleton<IServer, KestrelServer>();
                 });
             }
-            else if (string.Equals(webHost, "LinuxTransport", StringComparison.OrdinalIgnoreCase))
+            else
             {
-                builder.UseLinuxTransport(options =>
+                if (string.Equals(webHost, "Sockets", StringComparison.OrdinalIgnoreCase))
                 {
-                    options.ApplicationSchedulingMode = PipeScheduler.Inline;
-                });
+                    builder.UseSockets(options =>
+                    {
+                        if (int.TryParse(builder.GetSetting("threadCount"), out int threadCount))
+                        {
+                            options.IOQueueCount = threadCount;
+                        }
+#if NETCOREAPP5_0
+                    typeof(SocketTransportOptions).GetProperty("WaitForDataBeforeAllocatingBuffer")?.SetValue(options, false);
```

Comment:
@sebastienros @halter73 What's the deal with this? Don't we have the bits yet?

Reply:
The aspnet repos might not be up to date with the latest sdk. Alternatively we can force it in the driver command line.

Reply:
This was merged a while ago. It should work with a recent enough `$(MicrosoftAspNetCoreAppPackageVersion)`. We probably need to look at updating the csproj or dependencies.props.

Reply:
I set `--sdk 5.0.100-preview.4.20202.8` in the baseline KPI runs.

Let me try to update the sdk in the aspnet repos, I did it last time.